### PR TITLE
Create  How-to-log-life.md

### PR DESCRIPTION
**Pull request by marcobaturan**
This file can be modified by others or extend. PLease, review and critic It.

### CODE REVIEW

**Comment by ivan-karlovich** [comment]
I suggest that the file should be placed into the `docs` folder (the folder should be created) in order not to pollute the root folder. The content itself looks great. But probably it makes sense to provide links to the source of the information.

**Review by ivan-karlovich — APPROVED** [review]

**Comment by marcobaturan** [comment]
Ok, don't worry I will be crate a document folder and place the document
inside. Thank you very much for your advice.

File: How-to-log-life.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# How to download a copy of everything Facebook knows about you
```
Comment (ivan-karlovich) [inline How-to-log-life.md]: I guess this file can be deleted now as there is a copy in the `Documentation` folder.

**Comment by marcobaturan** [comment]
Thanks, I forget delete the file.

**Pull request merged**